libraries/AC_Avoidance/AC_Avoid.cpp

```diff
@@ -847,27 +888,34 @@ float AC_Avoid::distance_to_lean_pct(float dist_m)
     return 1.0f - (dist_m / _dist_max);
 }
 
-// returns the maximum positive and negative roll and pitch percentages (in -1 ~ +1 range) based on the proximity sensor
-void AC_Avoid::get_proximity_roll_pitch_pct(float &roll_positive, float &roll_negative, float &pitch_positive, float &pitch_negative)
+// returns the roll and pitch ratios (in -1 ~ +1 range) based on the proximity sensor
```

Comment:
Are they really "ratios" though?  when I read this it makes me think it's the ratio of roll to pitch.  I think they are actually roll and pitch angles expressed as a percentage from -1 to +1.